wallet/src/App.vue

```diff
@@ -80,27 +110,37 @@ export default defineComponent({
   },
   methods: {
     async connectWallet(e: any) {
-      switch (e.target.value) {
-        case WalletType.ALGOSIGNER:
-          this.connectAlgoSigner();
-          break;
-        case WalletType.MY_ALGO:
-          this.connectMyAlgoWallet();
-          break;
-        case WalletType.WALLET_CONNECT:
-          this.connectWalletConnect();
-          break;
-        default:
-          console.warn("Wallet %s not supported", e.target.value);
+      if (e.target.value) {
+        let walletType = e.target.value;
+        switch (walletType) {
+          case WalletType.ALGOSIGNER:
+            this.connectAlgoSigner();
+            break;
```

Comment:
```suggestion
          case WalletType.ALGOSIGNER: {
               this.connectAlgoSigner();
               break;
            }
```

Reply:
I think it more sense. However you also can skip this.

Reply:
Done